spec/formatting.md

```diff
@@ -428,6 +432,32 @@ These are divided into the following categories:
     > when * {The value is not one.}
     > ```
 
+  - **Variable Redefinition errors** occur when
+    the left-hand side of a variable declaration is already defined,
+    either by a previous local variable declaration,
+    or by an external definition.
+
+    > For example, attempting to format the following message
+    > must result in a Variable Redefinition error,
```

Comment:
must => MUST

... although examples are by definition non-normative, which means you should avoid normative language.... so...

must result => results